tests/test_end_to_end.py

```diff
@@ -135,6 +138,27 @@ def test_transform_col_query_wikisql(self):
         finally:
             self.spark.sql(f"DROP TABLE IF EXISTS {table_name}")
 
+    def test_filter_exact(self):
+        """Test that agent filters by an exact value"""
+        statements = create_temp_view_statements(
```

Comment:
Nit: we can have a function get_table_name
```
def get_table_name(tbl: str):
    statements = create_temp_view_statements("tests/data/test_transform.tables.jsonl")
    tbl_in_json = tbl.replaceAll("_", "-")
     for statement in statements:
            if tbl_in_json in statement:
                self.spark.sql(statement)
      return get_table_name(tbl)
```
So that we can reduce duplicated code and writing tests will be eaiser.